analyses/compare-gistic/run-compare-gistic.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Chante Bethell for CCDL 2019
+#
+# Run the GISTIC comparison R notebooks in this module sequentially.
+
+set -e
+set -o pipefail
+
+# This script should always run as if it were being called from
+# the directory it lives in.
+script_directory="$(perl -e 'use File::Basename;
```

Comment:
@jashapiro , what was the alternative you mentioned you had to this the other day?

Reply:
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/7442e280a206f844230fd2873ba6dbea98f95320/analyses/run-gistic/run-gistic-module.sh#L7